css/index.css

```
@@ -1,3 +1,284 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+body {
+	margin: 0px auto 0px auto;
+	width: 880px;
+	font-size: 16px;
+}
+header {
+	display:inline-block;
+	width: 100%;
+	position: relative;
+	padding: 15px;
+	background-color: rgb(238, 235, 235);
+}
+.navigation {
+	width: 880px;
+	top: 0px;
+	position: fixed;
+	z-index: 2px;
+}
+header nav a {
+	display: inline-block;
+	margin: 10px 28px 10px 28px;
+	text-decoration: none;
+	color: gray;
+	font-size: 16px;
+	transition: width 1s, height 1s, transform 1s;
+	transition: width 1s, height 1s, transform 1s;
+}
+header nav a:hover {
+	background-color: gray;
+	color: white;
+}
+header nav a:first-of-type {
+	margin-left: 0px;
+}
+header img {
+	position: absolute;
+	right: 0;
+}
```

```suggestion
header img {
	position: absolute;
	right: 0;
}
``` if you use flex on the parent you can move these with responsiveness what you are doing sets it to be at a specific spot at all times